cmd/panacead/cmd/genoracle.go

```diff
@@ -0,0 +1,225 @@
+package cmd
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	oracletypes "github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagOracleUniqueID      = "oracle-unique-id"
+	flagOraclePublicKey     = "oracle-public-key"
+	flagOracleRemoteReport  = "oracle-remote-report"
+	flagOraclePublicKeyPath = "oracle-public-key-path"
+
+	flagOracleAccount  = "oracle-account"
+	flagOracleEndpoint = "oracle-endpoint"
+	flagOracleCommRate = "oracle-commission-rate"
+)
+
+type OraclePubKeyInfo struct {
+	PublicKeyBase64    string `json:"public_key_base64"`
+	RemoteReportBase64 string `json:"remote_report_base64"`
+}
+
+func AddGenesisOracleCmd(defaultNodeHome string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add-genesis-oracle",
+		Short: "Add a genesis oracle and oracle module parameters to genesis.json",
+		Long: `
+			Using this command, you can set genesis oracle and oracle module parameters.
+			If you set oracle public key path for oracle module params, the flags for oracle public key and remote report would be ignored.
+
+			The desired format of oracle public key file is:
+
+			"public_key_base64" : <base64-encoded-oracle-public-key>
+			"remote_report_base64" : <base64-encoded-oracle-remote-report>
```

Comment:
We need to mention that we expect a JSON file.
```suggestion
			The desired format of oracle public key file is:

			{
				"public_key_base64" : "<base64-encoded-oracle-public-key>",
				"remote_report_base64" : "<base64-encoded-oracle-remote-report>"
			}
```